imap_processing/cdf/config/imap_codice_global_cdf_attrs.yaml

```
@@ -19,6 +19,34 @@ imap_codice_l1a_hskp:
     Logical_source: imap_codice_l1a_hskp
     Logical_source_description: IMAP Mission CoDICE Instrument Level-1A Housekeeping Data.
 
+imap_codice_l1a_hi_counters_aggregated:
+    <<: *instrument_base
+    Data_level: 1A
+    Data_type: L1A_hi-counters-aggregated->Level-1A Hi Aggregated Instrument Counts Data
+    Logical_source: imap_codice_l1a_hi-counters-aggregated
+    Logical_source_description: IMAP Mission CoDICE Hi Level-1A Aggregated Instrument Counts Data.
+
+imap_codice_l1a_hi_counters_singles:
+    <<: *instrument_base
+    Data_level: 1A
+    Data_type: L1A_hi-counters-aggregated->Level-1A Hi Single Instrument Counts Data
+    Logical_source: imap_codice_l1a_hi-counters-singles
+    Logical_source_description: IMAP Mission CoDICE Hi Level-1A Single Instrument Counts Data.
+
+imap_codice_l1a_hi_omni:
+    <<: *instrument_base
+    Data_level: 1A
+    Data_type: L1A_hi-omni->Level-1A Hi Omni Data
+    Logical_source: imap_codice_l1a_hi-omni
+    Logical_source_description: IMAP Mission CoDICE Hi Level-1A Omni Data.
```

I assume omni has an implied directional? Do we want to add that to the longer descriptors?
```suggestion
    Logical_source_description: IMAP Mission CoDICE Hi Level-1A Omnidirectional Data.
```